tests/unit/models/360-core.js

```diff
@@ -158,6 +158,28 @@ describe("SerSol 360 Core Model >", function() {
     });
   });
 
+  describe("serSol360Core model libraryIdOverride method staging >", function() {
+    beforeEach(function() {
+      delete browzine.api;
+    });
+
+    afterEach(function() {
+      delete browzine.libraryId;
+      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
+    });
+
+    it("should not override the libraryId on the api endpoint when the libraryId is not specified", function() {
+      browzine.api = "https://staging-api.thirdiron.com/public/v1/libraries/XXX";
```

Comment:
In both of the new tests, the libraryId is not specified, and we're expecting the same value on the same function result.  What is different about these two `it` statements?  I'm not quite telling the difference.